_posts/2016/2016-03-24-ecommerce-barcamp-jena.md

```diff
@@ -0,0 +1,56 @@
+
+## The participants
+
+The clientele consists mainly of UI and UX designers, frontend and backend developers, project managers, some marketing people and also a few dedicated testers from web agencies with 1 to 20 people, who typically work with self-contained content management systems like [Magento](https://magento.com), [OXID](http://www.oxid-esales.com), [Shopware](https://en.shopware.com), [TYPO3](https://typo3.org) or even [WordPress](https://wordpress.com). Attendees from the global eCommerce players are typically lesser represented, but this doesn't mean that local matadors like [Intershop](http://www.intershop.com) or ePages are not there as well.
+
+{% image blog/blog-ecommerce-barcamp-1.jpg %}{% endimage %}
+
+## The setting
+
+The event began with a dinner on Thursday evening in the famous local restaurant [Zur Noll](http://www.zur-noll.de/), which should break the ice of everyone's comfort zone and kickoff the communication flow.
+
+On Friday morning all participants met in the auditorium at the [Ernst-Abbe-Hochschule Jena](http://www.eah-jena.de/), University of Applied Sciences, which served as hosting space. After a collective breakfast the session planning for both days took place. Everybody can suggest an own Open Space talk. The crowd expresses interest in a suggestion via raising their hands. This immediate reaction allows a proper planning of the time slot and room allocation for the session.
+
+## The sessions
+
+To give you a better impression of the actual contents of the conference I want to outline some of the sessions. As I am a backend software craftsman you may recognise easily that I skipped the marketing and lawfully oriented parts of the event and tried to focus on the more technical talks.
+
+{% image blog/blog-ecommerce-barcamp-2.jpg %}{% endimage %}
+
+The first session was held by Christoph Rüger, who is part of a small team that has built a new cloud service that reads data from files (uploaded CSV, XML, Google Drive etc.) and let's the user create workflows via Drag'n'Drop (e.g. cronjobs with conditions and scheduling) that can interact with other service providers. Christopher revealed which tools his team used for internal project management and customer communication. He also explained the process of finding the fitting frameworks for developing their service, which was a quite interesting to follow.
```

Comment:
The first session was held by Christoph Rüger. He is part of a small team that has built a new cloud service that reads data from files (uploaded CSV, XML, Google Drive etc.). This service lets the user create workflows via drag and drop (e.g. cronjobs with conditions and scheduling) that can interact with other service providers. Christopher revealed, which tools his team used for internal project management and customer communication.